src/elements/chatbox/chatbox.js

```diff
@@ -18,14 +21,18 @@ export const ChatBox = ({ roomId }) => {
   }
 
   useEffect(() => {
-    console.log('Connecting new websocket to ' + url('api_ws.chatSocket', { args: { id: roomId } }))
-    socketRef.current = new window.WebSocket(
-      url('api_ws.chatSocket', { args: { id: roomId } })
-    )
+    // Alternative: put token in querystring:
+    // console.log('Connecting new websocket to ' + url('api_ws.chatSocket', { args: { id: roomId }, queries: { token: 'I AM TOKEN HI' }}))
+    const socketUrl = url('api_ws.chatSocket', { args: { id: roomId } })
+    console.log('Connecting new websocket to ' + socketUrl)
+    socketRef.current = new window.WebSocket(socketUrl)
 
     socketRef.current.onmessage = (e) => {
       const data = JSON.parse(e.data)
-      setChatLog((previous) => previous + data.message + '\n')
+      console.log('ONMESSAGE ACTIVATED WITH DATA: ')
+      console.log(JSON.stringify(data, null, 2))
```

Comment:
Just an FYI, you can do `console.log('ONMESSAGE:', { data })`

console methods accept any number of params, and if you pass an object literal, it will be explorable in the devtools.